### ISSUE TITLE
Tensorpack SyncMultiGPUTrainerParameterServer v.s. Pytorch DataParallel

### ISSUE THREAD

**Issue by iamhankai** [tensorpack/tensorpack#1419]
In SyncMultiGPUTrainerParameterServer, the gradients from multi-gpus are averaged, while in [DataParallel](https://pytorch.org/docs/stable/nn.html?highlight=dataparallel#torch.nn.DataParallel), they are summed. Right?
If I want to reproduce result from SyncMultiGPUTrainerParameterServer in Pytorch DataParallel, the learning rate in pytorch should be divided by #gpus?

**Comment by ppwwyyxx** [comment]
SyncMultiGPUTrainerParameterServer: see docs: https://tensorpack.readthedocs.io/modules/train.html#tensorpack.train.SyncMultiGPUTrainerParameterServer
DataParallel is unrelated to this project so we do not answer questions about it.
